dev/com.ibm.ws.microprofile.config.1.3/src/com/ibm/ws/microprofile/config13/sources/OSGiConfigUtils.java

```diff
@@ -178,22 +190,40 @@ public static String getCDIAppName(BundleContext bundleContext) {
     /**
      * Get the filter string which will extract <appProperties> elements for the given application
      *
-     * @param bundleContext The context to use in looking up OSGi service references
-     * @param applicationName The application name to look for
+     * @param applicationPid The pid of the application in the config
      * @return A filter string to be used by the OSGi ConfigurationAdmin service
      */
-    public static String getApplicationConfigFilter(BundleContext bundleContext, String applicationName) {
+    public static String getApplicationPropertiesConfigFilter(String applicationPid) {
         String applicationConfigFilter = null;
 
-        String applicationPID = getApplicationPID(bundleContext, applicationName);
+        StringBuilder applicationFilter = new StringBuilder(200);
+        applicationFilter.append("(&");
+        applicationFilter.append(FilterUtils.createPropertyFilter("service.factoryPid", "com.ibm.ws.appconfig.appProperties"));
```

Comment:
Ideally I think these strings should be extracted out into constants at the top of this class